intern/convenience/array.py

```diff
@@ -341,23 +342,31 @@ class AxisOrder:
     ZYX = "ZYX"
 
 
-class _MetadataProvider:
-    """
-    Serves as a dictionary-like API for resource metadata.
-
-    """
-
-    def __init__(self, dataset) -> None:
-        """
-        Create a new metadata provider.
-
-        Arguments:
-            dataset (array)
+class Metadata:
+    def __init__(self, resource: Union[BossResource, str], remote: BossRemote = None):
+        self._remote = remote or BossRemote(_DEFAULT_BOSS_OPTIONS)
+        if isinstance(resource, str):
+            resource = resource.split("://")[-1]
+            path_items = resource.split("/")
+            if len(path_items) == 1:
+                self._resource = self._remote.get_project(
+                    CollectionResource(path_items[0])
+                )
+            elif len(path_items) == 2:
+                self._resource = self._remote.get_project(
+                    ExperimentResource(path_items[1], path_items[0])
+                )
+            elif len(path_items) == 3:
+                self._resource = self._remote.get_project(
+                    ChannelResource(path_items[2], path_items[0], path_items[1])
+                )
+            else:
+                raise ValueError(f"Invalid resource path: {resource}")
+        else:
+            self._resource = resource
 
-        """
-        self._array = dataset
-        self._resource = dataset._channel
-        self._remote = dataset.volume_provider.get_remote()
+    def __truediv__(self, path: str):
```

Comment:
this is overloading the div?  Is the the way to list all collections?

Reply:
yeah, this is an overload of the `/` operator a la `pathlib.Path` (`Path("~") / "Documents" / "Rabbit Pictures"`)

This is not super important, just how I create a new `Metadata` object for path-accessed projects:

```python
m = Metadata("collection") / "experiment"

# is the same as:
m = Metadata("collection/experiment")
```